apps/storybook/src/Html.stories.tsx

```diff
@@ -201,10 +203,44 @@ export const Portal = {
               customContainer,
             )}
         </Html>
+        <Html>
+          {visCanvas &&
+            createPortal(
+              <div
+                style={{
+                  gridArea: 'canvas',
+                  justifySelf: 'start',
+                  alignSelf: 'end',
+                  width: '40em',
+                  padding: '0 1rem',
+                  border: '3px solid magenta',
+                  backgroundColor: 'rgba(255, 255, 255, 0.8)',
+                }}
+              >
+                <p>
+                  Using a portal, you can also render HTML elements into{' '}
+                  <strong>
+                    <code>visCanvas</code>
+                  </strong>
+                  , the root <code>div</code> rendered by <code>VisCanvas</code>
+                  , which includes the axes. You can then take advantage of the
+                  CSS Grid defined on <code>visCanvas</code> to position your
+                  elements.
+                </p>
+                <p>
+                  However, note that your elements will interfere with canvas
+                  interactions (zoom, pan, etc.), since pointer events will no
+                  longer be able to bubble up to <code>canvasArea</code>. You
+                  can remedy this with <code>pointer-events: none</code>.
```

Comment:
While I'm at it, adding a bit more documentation here hinting at how interactions work under the hood.